src/backend/distributed/utils/metadata_cache.c

```diff
@@ -668,6 +669,16 @@ DistNodeRelationId(void)
 }
 
 
+/* return oid of pg_dist_colocation relation */
+Oid
+DistColocationRelationId(void)
+{
```

Comment:
On cache invalidation, we need to set distColocationRelationId to invalidOid.
See `InvalidateDistRelationCacheCallback()` function.